pages/awards/community-awards.md

```diff
@@ -9,6 +9,14 @@ menubar_toc: true
 
 ## Community Awards
 
+### Year 2024 Community Awards Winners
+**Congratulations to our outstanding winners!**
+* Impact Award: [Winner's Name] for their exceptional contributions to the Research Software Engineer community.
+* Excellence in Service Award: [Winner's Name] for their dedication and commitment to the US-RSE organization and community.
+
+
+Learn more about our winners and their achievements [HERE](/2024-29-05-community-awards-results/).
```

Comment:
You should change the reference to the file here:
`2024-29-05-community-awards-results` -> `2024-05-29-community-awards-results`

Also, it would be better if you put that change from #1491 into this PR so that the preview can be generated and viewed correctly.  There's no connection between the two changes if you leave them as separate PRs.

Reply:
Changed the reference link. Cannot move PRs at this point.